README.md

```diff
@@ -1,21 +1,25 @@
 jsapi-resources
 ===============
-A collection of resources for developers using the [ArcGIS API for JavaScript](https://js.arcgis.com).
+A collection of resources for developers using the [ArcGIS API for JavaScript](https://js.arcgis.com) to create local builds. For more information, visit the [Developer Tooling](https://developers.arcgis.com/javascript/latest/guide/tooling-intro/) guide topic in the SDK.
```

Comment:
Not sure I'd add "to create local builds" since you may be getting the OAuth callback or using TypeScript with the CDN.

Reply:
True. Hmmm, okay this is easy to remove.

Reply:
Done.

Reply:
I still see "to create local builds"

Reply:
Oops, okay "this" time I got all of it.